api_specifications/case-api-0.1.0.yaml

```diff
@@ -399,9 +394,11 @@ components:
             $ref: '#/components/schemas/AdvocateMapping'
         status:
           type: string
-          description: >-
-            The current status of the case as pulled from the Workflow service.
-            Workflow service will hold the entire history of hte case.
+          description: The current status of the case as pulled from the Workflow service. Workflow service will hold the entire history of the case.
+        outcome:
+          type: string
+          format: uuid
```

Comment:
Is uuid format correct? or even needed here?

Reply:
yes. all master data is linked with a master data UUID